zilliqa/src/exec.rs

```diff
@@ -103,6 +105,8 @@ pub struct TransactionApplyResult {
     pub traces: Arc<Mutex<LoggingEventListener>>,
     /// The gas paid by the transaction
     pub gas_used: u64,
+    /// The scilla events, if any
+    pub scilla_events: Value,
```

Comment:
Should we attempt to decode the scilla events, rather than just leaving them as a JSON `Value`? IIRC we need to transform each one to a special representation when returning them from `eth_getLogs` so it might be easier to store something more structured internally?

Reply:
I could map them into eth `Log` object if that is what you mean? Is this for otterscan? I had assumed that scilla TXs are assumed not to have logs via the eth api

Reply:
Scilla events need to be encoded in EVM receipts as described in Section 5 of [RFC075](https://docs.google.com/document/d/196FM_IWCeKqBxKubii3b0af7veVPMV_r_i2K9q5myh8/edit?usp=sharing). ZQ1 implements this.